css/src/05taskbody.scss

```diff
@@ -70,19 +70,19 @@
 	}
 
 	.duedate {
-		color: $blue_due;
+		color: $color-primary;
```

Comment:
Same goes for ```$blue_due```, replacing it with ```$color_primary``` does not make sense, since it is matched to the icon color.